src/Tests/RegistrationTest.php

```diff
@@ -28,9 +28,36 @@ class RegistrationTest extends WebTestBase {
    * Test the standard registration process when there is no existing user.
    */
   public function testNormalNone() {
-    // @todo: Change the site config to disable acquisitions.
-
-    // @todo: Test when there is no user (should create).
+    $config = $this->config('user.settings');
+    // Don't require email verification and allow registration by site visitors
+    // without administrator approval.
+    $config
+      ->set('verify_mail', FALSE)
+      ->set('register', USER_REGISTER_VISITORS)
+      ->save();
+
+    // Disable acquisition on registration.
+    $acquisition_config = $this->config('decoupled_auth.settings');
```

Comment:
This could go in a protected method which gets called from the relevant test methods, save duplicating the code?